src/assets/images/landing_new/icon1.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="57" height="71" viewBox="0 0 57 71" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Since you replaced a bunch of the landing page images, can you delete the outdated ones from the repo that are not used anymore?